x-pack/plugins/inference/common/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export {
```

Comment:
What? So you mean you're aware that `index` files are supposed to be used for re-exporting, not living alone in their folder containing concrete stuff?

Reply:
who'd have thought!